src/test/java/io/tackle/pathfinder/services/AssessmentSvcTest.java

```diff
@@ -684,7 +684,7 @@ public void given_TwoAssessments_when_RequestedIdentifiedRisksInSpanishAndNullLa
         option4.selected = true;
 
         List<RiskLineDto> riskLineDtos = assessmentSvc.identifiedRisks(List.of(7766L, 8877L), "");
-        List<RiskLineDto> riskLineDtosES = assessmentSvc.identifiedRisks(List.of(7766L, 8877L), "ES");
+        List<RiskLineDto> riskLineDtosES = assessmentSvc.identifiedRisks(List.of(7766L, 8877L), "CA");
```

Comment:
Should this variable be renamed riskLineDtosCA?

Reply:
changed as suggested